ci/jenkins/jobs/macros.yaml

```diff
@@ -275,3 +275,55 @@
           set -e 
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test-mc.sh --testcase e2e --registry ${DOCKER_REGISTRY} --mc-gateway --codecov-token "${CODECOV_TOKEN}" --coverage --kind
+- builder:
+    name: builder-ipv6-e2e-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          ./ci/jenkins/test.sh --testcase e2e --registry ${DOCKER_REGISTRY} --testbed-type jumper
```

Comment:
According to the issue description, do you mean the old job definitions are incorrect and inconsistent with Jenkins?
If it's the case, I feel you'd better to make the description in the PR summary and commit message clearer. Maybe:
```
Change IPv6 and dual-stack Jenkins job script from test-vms.sh to test.sh to correct the
inconsistent definitions with the actual job definitions in the Jenkins.
```

Reply:
Changed the commit message and description.